extensions/amp-bind/0.1/bind-expression.js

```diff
@@ -139,23 +139,24 @@ export class BindExpression {
         const isBuiltIn = (args[0] === null);
 
         const caller = this.eval_(args[0], scope);
+        const callerType = Object.prototype.toString.call(caller);
```

Comment:
This is only used in the `else` block below. We should keep vars within the minimal scope and avoid doing this work unnecessarily when the invocation is on a built-in function.